src/services/Migrations.ts

```diff
@@ -40,6 +45,151 @@ class Migrations {
       console.log(`collection ${CachedValidator} removed.`);
     });
   };
+
+  private static migrateTo7280 = async () => {
+    await Migrations.wrapMigration('7.28.0', async () => {
+      // we have different index and validator per network, so let's simply drop and rebuild
+      const [uniswapV3PoolModel, tokenModel] = await Promise.all([
+        getModelForClass(UniswapV3Pool),
+        getModelForClass(Token),
+      ]);
+
+      console.log('get UniswapV3Pool and Token models.');
+
+      const uniswapV3PoolMigration = getUniswapV3Migration(uniswapV3PoolModel);
+      const tokenMigration = getTokenMigration(tokenModel);
+
+      try {
+        await Promise.all([uniswapV3PoolMigration, tokenMigration]);
+      } catch (reason) {
+        throw new Error(`Migration 7.28.0 failed: ${reason}`);
+      }
+
+      console.log('Migration 7.28.0 finished');
+    });
+  };
+}
+
+async function getUniswapV3Migration(uniswapV3PoolModel: ModelType<UniswapV3Pool>) {
+  const uniswapV3Pools = [
+    {
+      chainId: 'rootstock',
+      protocol: 'uniswapV3',
+      token0: '0x542fDA317318eBF1d3DEAf76E0b632741A7e677d',
+      token1: '0xef213441A85dF4d7ACbDaE0Cf78004e1E486bB96',
+      fee: 500,
+      address: '0xd2ffe51ab4e622a411abbe634832a19d919e9c55',
+      liquidityActive: '17716643053084368348144',
```

Comment:
liquidityActive, liquidityLockedToken0, and liquidityLockedToken1 are hardcoded values and do not represent the value in the pool.